src/AccessibilityInsights/Modes/ConfigurationModeControl.xaml.cs

```diff
@@ -97,26 +96,6 @@ void UpdateSaveButtonState()
             }
         }
 
-        /// <summary>
-        /// Redirect to main window login 
-        /// </summary>
-        /// <param name="serverUrl"></param>
-        /// <param name="prompt"></param>
-        /// <param name="callback"></param>
-        void HandleLoginRequest(Uri serverUrl = null, bool prompt = true, Action callback = null)
```

Comment:
Are we removing the experience when there's no configuration we navigate to the connection tab?

Reply:
No we will still have it. The login now happens at startup.